Prefix_Sum/Prefix_Sum.c

```diff
@@ -0,0 +1,33 @@
+#include<stdio.h>
+#include<stdlib.h>
+#define MAX 1000
+
+int prefixsum[MAX];//array to store prefixsums
+
+/*function that calculates prefix sum by 
+forming the prefixsum array
+*/
+void calculate_prefix_sum (int arr[], int n)
+{
+	prefixsum[0] = arr[0]; //starts with the first element
```

Comment:
Please use 4 space indentation throughout the file.

Reply:
pls check commit fa06829